plugins/@grouparoo/braze/src/lib/test.ts

```diff
@@ -0,0 +1,13 @@
+import { TestPluginMethod } from "@grouparoo/core";
+import { connect } from "./connect";
+import { BrazeClient } from "./client/client";
+
+export const test: TestPluginMethod = async ({ appOptions }) => {
+  const client: BrazeClient = await connect(appOptions);
+  const templates = await client.templates.get();
+  const success = !!templates;
+  const message = success
+    ? `${templates.length} Braze email templates available`
```

Comment:
We can probably show something a little more relevant for the test (eg: they might only be doing push notification marketing with braze).  Is there an account ID or something like that?